src/functions/scanOrder/handler.ts

```diff
@@ -0,0 +1,32 @@
+import 'source-map-support/register';
+
+import Order from '@dbModel/tables/order';
+import scanOrder from './function';
+
+import { ValidatedEventAPIGatewayProxyEvent, middyfy, Response, AuthenticatedUser } from 'utilities-techsweave';
+import StatusCodes from 'http-status-codes';
+
+import schema from './schema';
+
+
+/*
+ * Remember: event.body type is the type of the instantiation of ValidatedEventAPIGatewayProxyEvent
+ * In this case event.body type is type of 'Order'
+*/
+const scanOrderHandler: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+    let res: Response<Order> = new Response<Order>();
+
+    try {
+        //const user: AuthenticatedUser = await AuthenticatedUser.fromToken(event.headers?.AccessToken);
+
+
+        const result = await scanOrder(event.body);
```

Comment:
```suggestion
        const result = await scanOrder(event.body, userId);
```
Same logic of get:
- If user is customer: he can see only orders that belong to him
- If user is vendor: he can see all orders

I suggest some ideas about how implement this logic.